core/src/main/java/com/amplifyframework/core/DeveloperMenuActivity.java

```diff
@@ -16,16 +16,65 @@
 package com.amplifyframework.core;
 
 import android.app.Activity;
+import android.content.Context;
+import android.hardware.SensorManager;
```

Comment:
Still sort of "leaking" a detail of the `ShakeDetector`, here. I'd grab this off the `context `internally.